src/client/opamAdminCommand.ml

```diff
@@ -830,6 +830,100 @@ let check_command cli =
   Term.(const cmd $ global_options cli $ ignore_test_arg $ print_short_arg
         $ installability_arg $ cycles_arg $ obsolete_arg)
 
+module Compare_version_operator = struct
+  type t =
+    [ `Lt
+    | `Let
+    | `Eq
+    | `Gt
+    | `Get
+    ]
+
+  let all = [ `Lt ; `Let ; `Eq ; `Gt ; `Get ]
+
+  let eval : t -> int -> int -> bool = function
+    | `Lt -> ( < )
+    | `Let -> ( <= )
+    | `Eq -> ( = )
+    | `Gt -> ( > )
+    | `Get -> ( >= )
+
+  let assert_ t result = eval t result 0
+
+  let to_string : t -> string = function
+    | `Lt -> "<"
+    | `Let -> "<="
+    | `Eq -> "="
+    | `Gt -> ">"
+    | `Get -> ">="
+end
+
```

Comment:
The operator is already defined by `Cudf_types.relop` and mirrored in `OpamParserTypes.FullPos.relop_kind`, `OpamTypes` and `OpamFormula`. I think those operation should probably be defined in `OpamFormula` instead

Reply:
Done. Thanks!